__init__.py

```diff
@@ -47,6 +52,13 @@ class SystemCommand(Enum):
 
 
 class DeviceControlCenterSkill(NeonSkill):
+    user_config_path = expanduser("~/.config/neon/neon.yaml")
+
+    def initialize(self):
+        self.bus.on("mycroft.ready", self._speak_restart_dialog)
+        if self.dialog_to_speak and self.pending_audio_restart:
```

Comment:
If this happens before the audio module is ready, I believe the dialog may be lost.. The `on("mycroft.ready")` should handle it in any case so this appears to be redundant anyways

Reply:
See above, audio should always be ready before skills load in a Mark 2, so this is the primary and mycroft.ready is the fallback. mycroft.ready hasn't always been consistent for me so that's why I made this primary